<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShutdownTimer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/Icons/Icons/main.png</normaloff>:/Icons/Icons/main.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>228</width>
    <height>261</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Timer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxHours">
         <property name="suffix">
          <string> h</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxMinutes">
         <property name="suffix">
          <string> min</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSeconds">
         <property name="suffix">
          <string> s</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonOk">
      <property name="text">
       <string>Schedule</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>265</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>125</y>
      <width>166</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionEspa_ol"/>
    </widget>
    <addaction name="actionCancel_shutdown"/>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionCancel_shutdown">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icons/Icons/cancel.png</normaloff>:/Icons/Icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancel shutdown</string>
   </property>
  </action>
  <action name="actionEspa_ol">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icons/Icons/spanish.png</normaloff>:/Icons/Icons/spanish.png</iconset>
   </property>
   <property name="text">
    <string>Espa√±ol</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/Icons/Icons/english.png</normaloff>:/Icons/Icons/english.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
